_data/course.yml

```diff
@@ -15,6 +15,7 @@ gradescope_201a: "https://www.gradescope.com/"
 perusall_link: "https://app.perusall.com/courses/ecs-201a-001-wq-2023/"
 
 154b_github_classroom_link: "https://github.com/ECS154B-WQ23"
+dino_cpu_link: "https://github.com/ECS154B-WQ23"
```

Comment:
Can you link to this repo instead?

https://github.com/jlpteaching/dinocpu

Reply:
Sure, I changed it :)